test/route.js

```diff
@@ -450,20 +450,20 @@ describe('Route', () => {
         expect(res.payload).to.contain('hapi');
     });
 
-    it('throws when server timeout is more then socket timeout', () => {
+    it('allows server timeout more then socket timeout', () => {
```

Comment:
We should probably have similar tests for the payload timeout.

Reply:
Done!